cloudinary-image-management-and-manipulation-in-the-cloud-cdn/js/src/components/notices.js

```diff
@@ -0,0 +1,36 @@
+/* global window wp  */
+const Notices = {
+    _init: function() {
+        let self = this;
+        if ( typeof CLDIS !== 'undefined' ) {
+            let triggers = document.getElementsByClassName( 'cld-notice' );
+            [ ...triggers ].forEach( ( trigger ) => {
```

Comment:
For clarity, I suggest we name these variables notices/notice:

```suggestion
            let notices = document.getElementsByClassName( 'cld-notice' );
            [ ...notices ].forEach( ( notice ) => {
```